nav2_controller/nav2_controller/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>nav2_controller</name>
+  <version>0.0.1</version>
+  <description>
+    ROS2 controller (DWB) metapackage
+  </description>
+  <maintainer email="oregon.robotics.team@intel.com">Oregon Robotics Team</maintainer>
```

Comment:
Let's remove any references to this ficticious e-mail address and use specific addresses. For this one, we can use Carl's info instead (along with steve's).

Reply:
Will be in next PR